### DISCUSSION TITLE
Update data via csv

### DISCUSSION THREAD

**Discussion by yewkay** [Budibase/budibase#7353] [Ideas]
It seems that there is only an import function in the database CSV import function. At the moment I didn't find a function within budibase to update a table with existing data using csv. Importing a csv into a table with data will repeat the data. 

I would like to propose a way of matching the data to be updated eg: using a user-defined column as the ID, so each row item in that column will be matched and all the other columns will be updated with the new CSV being imported.

Additional suggestions to build on the import function:
- it could also contain mapping of columns, eg: DB columns A, B, C.. etc are matched to CSV's Columns E,F,G
- Additional columns that are not matched in the CSV can have the option to either ignore or create a new column

**Comment by melohagan** [comment]
These are great ideas! @yewkay 

I've raised a feature request here: https://github.com/Budibase/budibase/issues/7400
